sysconfig/cloudinit.go

```diff
@@ -151,9 +151,13 @@ var (
 	nocloudRestrictYaml = []byte(`datasource_list: [NoCloud]
 datasource:
   NoCloud:
-    fs_label: null`)
+    fs_label: null
+manual_cache_clean: true
```

Comment:
We should probably leave a reference to the bug itself and https://cloudinit.readthedocs.io/en/latest/topics/boot.html?highlight=manual_cache_clean#first-boot-determination right above `nocloudRestrictYaml`

Reply:
agreed on both!

Reply:
added